modules/apps/commerce/commerce-product-service/src/main/java/com/liferay/commerce/product/service/impl/CPInstanceLocalServiceImpl.java

```diff
@@ -1046,6 +1055,9 @@ public CPInstance updateCPInstance(
 
 		_validateSku(cpInstance.getCPDefinitionId(), cpInstanceId, sku);
 
+		_validateSkuNumericValues(
```

Comment:
let's call this only _validate and move this as first (and remove empty lines between the _validate* statements)